src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/ObjectiveC/TrackedNativeReferenceAttribute.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Versioning;
+
+namespace System.Runtime.InteropServices.ObjectiveC
+{
+    /// <summary>
+    /// Attribute used to indicate a class is tracked from the native environment.
+    /// </summary>
+    [SupportedOSPlatform("macos")]
```

Comment:
not an immediate concern, it's a few steps ahead, but we'll need `maccatalyst` too to support MAUI (and likely Blazor) too
https://github.com/dotnet/designs/pull/174